docs/gathered-data.md

```diff
@@ -1108,7 +1109,7 @@ None
 - `console_helm_installs_total` introduced in version 4.11+
 - `console_helm_upgrades_total` introduced in version 4.12+
 - `console_helm_uninstalls_total` introduced in version 4.12+
-- `openshift_apps_deploymentconfigs_strategy_total` introduced in version 4.13+
+- `openshift_apps_deploymentconfigs_strategy_total` introduced in version 4.12.5+, 4.13+
```

Comment:
I think that here we should say `... introduce in version 4.13+ and backport to 4.12.5+` - just to be clear

Reply:
Well we had:

Reply:
Ouch! And this is probably my bad. This is not good as well. Do you mind updating all? 🥺